public/js/attendance-scripts/attendance-utilities.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * PEP Capping 2017 Algozzine's Class
+ *
+ * javascript utility functions
+ *
+ * functions used to reduce duplicated code
+ *
+ * @author Scott Hansen
+ * @copyright 2017 Marist College
+ * @version [version number]
+ * @since [initial version number]
+ */
+
+function populateTimes() {
+    var selectTime = document.getElementById('time-input');
+
+    var optionArray = [];
```

Comment:
Remove this unused array. You're appending directly to the `time-input` element now.